spec/coursemology/evaluator/models/base_spec.rb

```diff
@@ -0,0 +1,37 @@
+RSpec.describe Coursemology::Evaluator::Models::Base do
+  describe 'class attributes' do
+    subject { Coursemology::Evaluator::Models::Base }
+
+    describe '.api_user_email' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.